package.json

```diff
@@ -57,6 +57,7 @@
         "@babel/parser": "^7.26.2",
         "@babel/traverse": "^7.25.9",
         "@babel/types": "^7.26.0",
+        "@prettier/plugin-oxc": "^0.0.4",
```

Comment:
Is there some way to make this an optional dependency?

Feels like we should only integrate this plugin *if* the user already has this plugin installed.

Reply:
Maybe a function with an inline require? Only should be executed if the processor gets triggered. I'm fine if that has a `// ts-ignore` + maybe a try/catch to log a warning the user if they forgot to install a plugin, but prettier is trying to use it?

Reply:
Let me know if it's OK like that